llama_stack/templates/together/together.py

```diff
@@ -81,14 +81,22 @@ def get_distribution_template() -> DistributionTemplate:
             provider_id="code-interpreter",
         ),
     ]
-    embedding_model = ModelInput(
-        model_id="all-MiniLM-L6-v2",
```

Comment:
We can still dynamically register `all-MiniLM-L6-v2` with `sentence-transformers` provider.